gulpfile.js

```diff
@@ -106,9 +106,11 @@ var cssFilePaths = [];
 var jsFilePaths = [];
 var fontFolderPaths = [];
 var cssBackgroundFilepaths = [];
+// There is no cache slug for dev.
+var cacheSlugDev = '';
 var generatedTargetDir = path.join(
   'third_party', 'generated',
-  isMinificationNeeded ? 'prod' : 'dev');
+  isMinificationNeeded ? 'prod' : cacheSlugDev);
```

Comment:
I'm confused about this. This refers to the files generated in /third_party/generated. I don't see why anything needs to change here, nor what it has to do with the cache slug?